imagetest/test_suites/oslogin/ssh_test.go

```diff
@@ -222,8 +227,8 @@ func TestSSH(t *testing.T) {
 	}
 }
 
+// Checks if an admin user can use sudo after successfully SSH-ing to an instance.
 func TestAdminSSH(t *testing.T) {
-	time.Sleep(20 * time.Second)
```

Comment:
why is this sleep not relevant any more?

Reply:
The tests are run in sequence. So if the TestAgent test is already done by the time the first test `TestSSH` runs, then there's no more need to wait for TestAgent to finish if it's already done.

Reply:
I will document this in a followup commit.